examples/draft-0-10-0/plaintext/plaintext.html

```diff
@@ -49,7 +51,7 @@
                   editorState={this.state.editorState}
                   onChange={this.onChange}
                   placeholder="Enter some text..."
-                  ref="editor"
+                  ref={(d)=>{this.domEditor = d}}
```

Comment:
This will create a new arrow function on every render, which is unnecessary. Instead, do:
```
ref={this.setDomEditorRef}
```

Then add a `setDomEditorRef` method. Looks like it does this in the constructor so it's bound correctly:
```
this.setDomEditorRef = ref => this.domEditor = ref;
```